src/graphql/mirror.test.js

```diff
@@ -3377,24 +3377,56 @@ describe("graphql/mirror", () => {
         const db = new Database(":memory:");
         const mirror = new Mirror(db, buildGithubSchema());
 
+        type RawType = {|+typename: Schema.Typename, +id: Schema.ObjectId|};
+
         const objects = {
-          repo: () => ({typename: "Repository", id: "repo:foo/bar"}),
-          issue1: () => ({typename: "Issue", id: "issue:#1"}),
-          issue2: () => ({typename: "Issue", id: "issue:#2"}),
-          issue3: () => ({typename: "Issue", id: "issue:#3"}),
-          alice: () => ({typename: "User", id: "user:alice"}),
-          bob: () => ({typename: "User", id: "user:bob"}),
-          ethereal: () => ({typename: "User", id: "user:ethereal"}),
-          nobody: () => ({typename: "User", id: "user:nobody"}),
-          noboty: () => ({typename: "Bot", id: "bot:noboty"}),
-          comment1: () => ({typename: "IssueComment", id: "comment:#2.1"}),
-          comment2: () => ({typename: "IssueComment", id: "comment:#2.2"}),
-          closedEvent: () => ({
+          repo: (): RawType => ({
```

Comment:
Can we type the object literal as `{+[string]: () => RawType}` instead of
touching all these lines? That way, the diff is smaller, the blame is
higher-fidelity, and the code itself is easier to read.